ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -339,7 +339,7 @@ Get Model Inference
             ${rc}    ${url}=    Run And Return Rc And Output
             ...    oc get ksvc ${model_name}-predictor -n ${project_title} -o jsonpath='{.status.url}'
             Should Be Equal As Integers    ${rc}    0
-            ${curl_cmd}=    Set Variable    curl -s ${url}${end_point} -d ${inference_input}
+            ${curl_cmd}=    Set Variable    curl -ks ${url}${end_point} -d ${inference_input}
```

Comment:
we should already have the logic to retrieve the certificate instead of using "-k"  flag. Could you try using that?